docs/source/03_tutorial/06_visualise_pipeline.md

```diff
@@ -96,3 +96,72 @@ kedro viz --load-file my_shareable_pipeline.json
 ```
 
 And this will visualise the pipeline visualisation saved as `my_shareable_pipeline.json`.
+
+
+## Visualise Plotly charts in Kedro-viz
+
+[Plotly](https://plotly.com/python/) is a free and open source python library that allows you to make interactive, publication quality graphs. With Plotly integration on Kedro-viz, you can output your interactive charts as part of your pipline visualisation. 
+
+Kedro-viz aims to help users communicate different aspects of thier workflow and with Plotly intergration, we take one step further in this direction to allow our users share thier data insights as well. 
```

Comment:
```suggestion
Kedro-Viz aims to help users communicate different aspects of their workflow and with the Plotly integration, we take one step further in this direction and allow our users to share their data insights as well. 
```

Reply:
building on top of Merel's suggestion above: 
```suggestion
Kedro-Viz aims to help users communicate different aspects of their workflow through an interactive flowchart. With Plotly integration, we take one step further in this direction to allow our users to effectively share their data insights while exploring the pipeline.
```

Reply:
lol this is a childhood problem .. i can never spell 'their' 
The number of times my english teacher must have put a red circle on it >< haha